### PR TITLE
Attempting to repair npm published package

### LINKED ISSUE #6692: Nextjs: No way to call "consumeError"
**Describe the bug**
I'm following the guide and trying to use the app router with `serverActions`, but looks like by default, our Server actions errors are not being tracked.

We also can't use `consumeError` to catch the errors since `@highlight-run/next/server` doesn't include it and `@highlight-run/node` fails when being imported on NextJs (because of `ansi-color` using Octal escape characters)

### LINKED ISSUE #6746: Dynamic require of "async_hooks" is not supported
I've made the issue yesterday about `BatchSpanProcessorBase` – https://github.com/highlight/highlight/issues/6741

@deltaepsilon made the fix, I updated to 3.4.1, and now I'm getting the new error:

```
error - Error: Dynamic require of "async_hooks" is not supported
    at node_modules/@highlight-run/node/dist/index.mjs:12:9
    at node_modules/@opentelemetry/context-async-hooks/build/src/AsyncHooksContextManager.js (node_modules/@highlight-run/node/dist/index.mjs:17388:22)
    at __require2 (node_modules/@highlight-run/node/dist/index.mjs:18:51)
    at node_modules/@opentelemetry/context-async-hooks/build/src/index.js (node_modules/@highlight-run/node/dist/index.mjs:17511:19)
    at __require2 (node_modules/@highlight-run/node/dist/index.mjs:18:51)
    at node_modules/@opentelemetry/sdk-trace-node/build/src/NodeTracerProvider.js (node_modules/@highlight-run/node/dist/index.mjs:24089:33)
    at __require2 (node_modules/@highlight-run/node/dist/index.mjs:18:51)
    at node_modules/@opentelemetry/sdk-trace-node/build/src/index.js (node_modules/@highlight-run/node/dist/index.mjs:24144:19)
    at __require2 (node_modules/@highlight-run/node/dist/index.mjs:18:51)
    at node_modules/@opentelemetry/sdk-node/build/src/index.js (node_modules/@highlight-run/node/dist/index.mjs:90905:21)
    at __require2 (node_modules/@highlight-run/node/dist/index.mjs:18:51)
    at node_modules/@highlight-run/node/dist/index.mjs:120160:31
    at ModuleJob.run (node:internal/modules/esm/module_job:193:25)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:530:24)
    at async importModuleDynamicallyWrapper (node:internal/vm/module:438:15)
```

To Reproduce

1. npm install --save @highlight-run/node ("@highlight-run/node": "3.4.0",)
2. import { H } from '@highlight-run/node';
3. H.init({ projectID: '***' });
4. Getting the error

### LINKED ISSUE #6747: Dynamic require of "async_hooks" is not supported
Greetings.

I requested a problem yesterday regarding BatchSpanProcessorBase - [#6741.](https://github.com/highlight/highlight/issues/6741)

@deltaepsilon fixed it, I updated to 3.4.1, but now I'm getting a new error:

```
error - Error: Dynamic require of "async_hooks" is not supported
    at node_modules/@highlight-run/node/dist/index.mjs:12:9
    at node_modules/@opentelemetry/context-async-hooks/build/src/AsyncHooksContextManager.js (node_modules/@highlight-run/node/dist/index.mjs:17388:22)
    at __require2 (node_modules/@highlight-run/node/dist/index.mjs:18:51)
    at node_modules/@opentelemetry/context-async-hooks/build/src/index.js (node_modules/@highlight-run/node/dist/index.mjs:17511:19)
    at __require2 (node_modules/@highlight-run/node/dist/index.mjs:18:51)
    at node_modules/@opentelemetry/sdk-trace-node/build/src/NodeTracerProvider.js (node_modules/@highlight-run/node/dist/index.mjs:24089:33)
    at __require2 (node_modules/@highlight-run/node/dist/index.mjs:18:51)
    at node_modules/@opentelemetry/sdk-trace-node/build/src/index.js (node_modules/@highlight-run/node/dist/index.mjs:24144:19)
    at __require2 (node_modules/@highlight-run/node/dist/index.mjs:18:51)
    at node_modules/@opentelemetry/sdk-node/build/src/index.js (node_modules/@highlight-run/node/dist/index.mjs:90905:21)
    at __require2 (node_modules/@highlight-run/node/dist/index.mjs:18:51)
    at node_modules/@highlight-run/node/dist/index.mjs:120160:31
    at ModuleJob.run (node:internal/modules/esm/module_job:193:25)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:530:24)
    at async importModuleDynamicallyWrapper (node:internal/vm/module:438:15)
```

To Reproduce

1. npm install --save @highlight-run/node ("@highlight-run/node": "3.4.0",)
2. import { H } from '@highlight-run/node';
3. H.init({ projectID: '***' });
4. Getting the error

### PR DESCRIPTION
**Pull request by deltaepsilon**
- Repairing the client import by publishing `client.d.ts`
- Repair `clean-dist.sh`
- Internalize `@highlight-run/sourcemap-uploader`
- Downgrade `@opentelemetry/api`

closes #6747 closes #6692

### CODE REVIEW

**Comment by farhan787** [comment]
Hi @deltaepsilon, I was facing and then tried to fix issue-6692 but I was struggling a bit with how to test the sdk code changes. Would be really helpful if you can redirect me to some doc (if exist).

File: sdk/highlight-next/bin/clean-dist.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
-sed -i '' '/"use client";/d' dist/next-client.js
-sed -i '' '/"use client";/d' dist/next-client.cjs
+sed -i '/"use client";/d;' dist/next-client.js
+sed -i '/"use client";/d;' dist/next-client.cjs
```
Comment (Vadman97) [inline sdk/highlight-next/bin/clean-dist.sh:2]: confirmed this is working correctly

File: sdk/highlight-next/package.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-	"version": "4.4.1",
+	"version": "4.4.2",
 	"description": "Client for interfacing with Highlight in next.js",
 	"files": [
 		"dist",
-		"next-client.d.ts",
+		"client.d.ts",
 		"server.d.ts",
-		"client.d.ts"
+		"types.d.ts"
```
Comment (Vadman97) [inline sdk/highlight-next/package.json:9]: i don't see a `client.d.ts`, only `next-client.d.ts` - is that what you meant?
```bash
 ~/w/highlight   vadim/fix-error-session-filter   ls sdk/highlight-next/dist/*.d.ts
sdk/highlight-next/dist/next-client.d.ts                   sdk/highlight-next/dist/server.edge.d.ts
sdk/highlight-next/dist/server.d.ts                        sdk/highlight-next/dist/with-highlight-edge-23c1ab0e.d.ts
```

**Review by Vadman97 — APPROVED** [review]

File: sdk/highlight-next/package.json (hunk 2 above)
Comment (deltaepsilon) [reply sdk/highlight-next/package.json:9]: No. I have a `client.d.ts` file that references `dist/next-client.d.ts`. For some reason I can't have a root file named `client.ts`. It was throwing weird errors. So I prefixed the actual file with `next-` and then aliased to it in `package.json#exports`. Of course, I can have a `client.d.ts` file 🤷

Comment (Vadman97) [reply sdk/highlight-next/package.json:9]: ah that's my bad, forgot the path here
```bash
 ~/w/highlight   vadim/fix-error-session-filter   ls sdk/highlight-next/client.d.ts
sdk/highlight-next/client.d.ts
```

**Comment by deltaepsilon** [comment]
> Hi @deltaepsilon, I was facing issue 6692 while integrating highlight in personal next.js project and then tried to fix it in highglight repo but I was struggling a bit with how to test the sdk code changes. Would be really helpful if you can redirect me to some doc (if exist). Apologies to comment on this PR.

I have a project in the Highlight monorepo that you can launch with `yarn e2e:nextjs`. That's where I test everything first. If that works, I then deploy it to Vercel and check that it works in Edge and Vercel's Node.js runtime. Vercel can be quirky.

I run a lot of `yarn build:sdk` and `yarn e2e:nextjs`. Then I open up `localhost:3005` and fire a ton of errors and make sure that they flow through to `app.highlight.io`.

This latest batch of bugs comes from testing in the monorepo. The monorepo has access to all sorts of dev dependencies that don't get published, and they were masking some problems. So now I have a third step, which is to pull it into an unrelated project after publishing and test again. This PR fixes the issues that I found in the unrelated project.

**Pull request merged**